### PR TITLE
172 category cruft

### LINKED ISSUE #172: Remove all category/severity cruft
Follow-up to #164.

We had left these in so we could run the old tests against the new code. However, we've decided not to worry about that. This means we can remove all of the references to category/severity.

### PR DESCRIPTION
**Pull request by linclark**
Fixes #172 
## Testing instructions

Everything should work the same. These are the tricky cases:
1. Make sure you can still filter `console.count()`
2. Make sure `console.info()` still displays icon

### CODE REVIEW

**Comment by linclark** [comment]
@nchevobbe since you've probably dealt with the severity/category stuff the most in the new console, would you mind giving this a review?

**Comment by linclark** [comment]
@bgrins you might want to give this a quick lookover too

File: devtools/client/webconsole/new-console-output/utils/messages.js [hunk 1]
```diff
@@ -138,6 +126,44 @@ function convertCachedPacket(packet) {
...
+  const levels = {
+    LEVEL_ERROR: "error",
+    LEVEL_WARNING: "warn",
+    LEVEL_INFO: "info",
+    LEVEL_LOG: "log",
+    LEVEL_DEBUG: "debug",
+  };
+
+  // A mapping from the console API log event levels to the Web Console
+  // levels.
```
Comment (nchevobbe) [inline devtools/client/webconsole/new-console-output/utils/messages.js]: Looks like this could be on the previous line since we have a 90 char limit now

**Comment by nchevobbe** [comment]
This looks good to me

File: devtools/client/webconsole/new-console-output/utils/messages.js (hunk 1 above)
Comment (linclark) [inline devtools/client/webconsole/new-console-output/utils/messages.js]: Good point, fixed

**Comment by linclark** [comment]
Great, thanks for the speedy review!

**Pull request merged**